### ISSUE TITLE
You can build drills you havent researched yet

### ISSUE THREAD

**Issue by MarcusSerenusSchneider** [Pointifix/AutoDrill#7]
![Screenshot 2023-05-04 203435](https://user-images.githubusercontent.com/131772666/236298269-07f4f44a-1310-427d-88cd-604f40f8ed2d.png)
This isnt bad but i thought i should report it

**Comment by Simon-VIRNECT** [comment]
Yeah I know someone already reported this, I will keep it in mind and maybe fix at some point but for now I'm fine with that. If you don't don't want to be able to do that just don't use the feature for now 😀

**Comment by Pointifix** [comment]
will not be considered
